src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -79,6 +79,7 @@ public class ViewResult : ActionResult
                             actionContext = context,
                             result = this,
                             viewName = viewName,
+                            viewEgineType = viewEngine.GetType(),
```

Comment:
Only found it today. I don't mind if it goes in post Beta8... I'm not trying to sneak anything past. Was going to talk to you about it this afternoon but you busy, so because it was a "simple" change I thought I would start the conversation with a PR.